### ISSUE TITLE
load path from the command line -I should use the same splitting as for GEM_PATH

### ISSUE THREAD

**Issue by mkristian** [jruby/jruby#3750]
line https://travis-ci.org/jruby/jruby/jobs/117553486#L4709
shows a forked jruby process by rspec/rake
```
java -cp :/home/travis/build/jruby/jruby/maven/jruby/target/it/runnable/runnable.jar org.jruby.Main -Iuri:classloader:/gems/rspec-core-3.4.3/lib:uri:classloader:/gems/rspec-support-3.4.1/lib uri:classloader://gems/rspec-core-3.4.3/exe/rspec --pattern spec/\*\*\{,/\*/\*\*\}/\*_spec.rb
```

the load-path should use the same regexp for splitting it as we use for GEM_PATH in rubygems.

### Environment

jruby-9.1.0.0-SNAPSHOT

### Expected Behavior

$LOAD_PATH should be ```['uri:classloader:/gems/rspec-core-3.4.3/lib']```

### Actual Behavior

$LOAD_PATH is ```['uri', 'classloader', '/gems/rspec-core-3.4.3/lib']```

**Comment by kares** [comment]
although `$LOAD_PATH` doesn't know anything about RubyGems and its spliting logic or does it?

**Comment by mkristian** [comment]
```$LOAD_PATH``` is an array. ```-Ipath1:path2``` commandline switch is the thing which needs the splitting logic

**Comment by headius** [comment]
It seems like this logic is there in ArgumentProcessor...

**Comment by mkristian** [comment]
was fixed by fa4846f2c12fef66b31f0973cc58c66fb30cb4fa
